app/javascript/components/zone-form/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import MiqFormRenderer from '@@ddf';
+import createSchema from './zone-form.schema';
+
+const ZoneForm = () => {
+
+    return (
+        <MiqFormRenderer
+            schema={createSchema()}
+            onSubmit={console.log}
+            onCancel={console.log}
+        />
+    );         
+};
+
+
```

Comment:
No need for the 2nd newline, one is enough to logically separate blocks of code.